tests/mobile_native/android_react_native/test_checkout_react_native_android.py

```diff
@@ -0,0 +1,24 @@
+def test_checkout_react_native_android(android_react_native_emu_driver):
```

Comment:
This isn't a 'new' file, I just moved the react native test contents out of `tests/mobile_native/android/test_checkout_android.py` and into a new file.